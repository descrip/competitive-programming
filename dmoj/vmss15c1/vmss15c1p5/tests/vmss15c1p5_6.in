10 934 965
applecomputer 869 18 9
asdfghjkl 910 15 21
ccc 69 26 10
cigarette 938 20 6
comsci 331 16 32
dp 199 11 28
ermergerd 321 23 23
partimejob 135 7 17
sucks 284 19 7
www 413 3 33
