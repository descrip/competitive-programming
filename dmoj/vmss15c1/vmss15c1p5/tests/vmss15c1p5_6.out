52249
applecomputer 1
asdfghjkl 41
ccc 0
cigarette 15
comsci 0
dp 0
ermergerd 0
partimejob 0
sucks 0
www 0
